### PR TITLE
[OTX] Fix logo path & codacy link

### PR DESCRIPTION
**Pull request by JihwanEom**
- Fix broken links for OTX logo, codacy links
- Fix a typo from `datumaro` to `Datumaro` for unified expression in README.md

### CODE REVIEW

**Review by sungmanc — CHANGES_REQUESTED** [review]
Could you leave a description about this PR?

File: README.md [hunk 1]
```diff
@@ -50,7 +50,7 @@ toolkit](https://software.intel.com/en-us/openvino-toolkit). For example, users
   - Update `find` command to find configurations of tasks/algorithms
   - Introduce `build` command to customize task or model configurations
   - Automatic algorihm selection for the `train` command using the given input dataset
-- Adaptation of [Datumaro](https://github.com/openvinotoolkit/datumaro) component as a dataset interface
+- Adaptation of [datumaro](https://github.com/openvinotoolkit/datumaro) component as a dataset interface
```
Comment (goodsong81) [inline README.md]: Do we need this change?
image

Comment (JihwanEom) [reply README.md]: Then how about to change `Datumaro` to`datumaro` in README.md?

Comment (goodsong81) [reply README.md]: @wonjuleee Could you give us some advice? `Datumaro` or `datumaro`?

Comment (goodsong81) [reply README.md]: > Then how about to change `Datumaro` for `datumaro` in README.md?

Sorry, I cannot understand. Do you mean that we need to replace all `datumaro` by `Datumaro`?

**Review by wonjuleee — APPROVED** [review]

**Review by sungmanc — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (JihwanEom) [reply README.md]: Wonju said `Datumaro` is right :)

**Review by goodsong81 — APPROVED** [review]

**Pull request merged**